reinforcement_learning/bindings/python/README.md

```diff
@@ -1 +1,14 @@
+# Python Bindings
+
+## Build + Install
+1. [Install pip for python3.6](https://askubuntu.com/questions/889535/how-to-install-pip-for-python-3-6-on-ubuntu-16-10)
+2. `sudo python3.6 -m pip install wheel`
+3. `sudo python3.6 -m pip install setuptools`
+4. Edit reinforcement_learning/bindings/python/setup.py to point to a directory on your machine (Will install dependencies here)
```

Comment:
We shouldn't be asking people to modify a file to build. Can we modify setup.py to accept a dirs. argument instead. We should also edit the current directory string '/home/ataymano/restore_e2e_1' to remove usernames.

Reply:
setup.py is called by the makefile which is called by the global vw makefile. So there isn't an easy way to parametize this. Best I can think of right now is a config file that this is picked up from?

Reply:
Went with an environment variable - see the latest update